charts/csi-isilon/Chart.yaml

```diff
@@ -1,20 +1,19 @@
 apiVersion: v2
 name: csi-isilon
-version: 2.10.1
-appVersion: "2.10.1"
-kubeVersion: ">= 1.23.0"
-#If you are using a complex K8s version like "v1.23.3-mirantis-1", use this kubeVersion check instead
-#WARNING: this version of the check will allow the use of alpha and beta versions, which is NOT SUPPORTED
-#kubeVersion: ">= 1.23.0-0"
+version: 2.11.0
+appVersion: "2.11.0"
+kubeVersion: ">= 1.21.0"
```

Comment:
Is this right, that we're going to support even older versions of k8s than before?

Reply:
This was discussed with POs, needs a broader discussion with PDMs before revving up. We can leave it as is now.